fe/src/main/java/org/apache/doris/load/loadv2/JobState.java

```diff
@@ -21,6 +21,7 @@
 public enum JobState {
     UNKNOWN, // this is only for ISSUE #2354
     PENDING, // init state
+    ETL,     // load data partition, sort and aggregation with etl cluster
```

Comment:
Is this OK to put a new State in the middle of original states?

Reply:
JobState will be persisted in meta data by name, so the order of these state is not important